compiler/src/test/java/com/squareup/anvil/compiler/dagger/ProvidesMethodFactoryGeneratorTest.kt

```diff
@@ -2249,6 +2250,28 @@ public final class DaggerComponentInterface implements ComponentInterface {
     }
   }
 
+  @Test fun `a return type for a provider method is required`() {
```

Comment:
I love these kinds of integration tests. So easy to read and comprehend.

Reply:
😇